### PR TITLE
Feature: enable filebrowser in readonly folder

### PR DESCRIPTION
**Pull request by lizable**
## Description
This PR enables filebrowser execution not only in writable folders but also in "read-only" folders.

> ⚠️ NOTE:
> Filebrowser button is enabled ONLY under conditions:
> - [x] : Filebrowser environment is already installed at least one or more agents.
> - [x] : Enough resources to execute session (**minimum resource to create the session: 1 CPU Core, 0.5 RAM**)

## Before
filebrowser-disabled-readonly

## After
filebrowser-enabled-readonly

Since users can't write any files or folders in read-only folders, uploading is not allowed likewise.
Therefore I added a notification dialog when executing filebrowser in folder-explorer dialog of read-only folder so that the user could understand the situation without facing errors.
notice-filebrowser

### CODE REVIEW

File: src/components/backend-ai-storage-list.ts [hunk 1]
```diff
@@ -277,6 +281,10 @@ export default class BackendAiStorageList extends BackendAIPage {
           --mdc-icon-button-size: 28px;
         }
 
+        #filebrowser-notificatin-dialog {
```
Comment (adrysn) [inline src/components/backend-ai-storage-list.ts]: typo: `notification` -> `notification`.

File: src/components/backend-ai-storage-list.ts [hunk 2]
```diff
@@ -965,6 +972,19 @@ export default class BackendAiStorageList extends BackendAIPage {
           </mwc-button>
         </div>
       </backend-ai-dialog>
+      <backend-ai-dialog id="filebrowser-notificatin-dialog" fixed backdrop narrowLayout>
```
Comment (adrysn) [inline src/components/backend-ai-storage-list.ts]: same here.

File: src/components/backend-ai-storage-list.ts [hunk 3]
```diff
@@ -2508,6 +2528,10 @@ export default class BackendAiStorageList extends BackendAIPage {
   _executeFileBrowser() {
     if (this._isResourceEnough()) {
       if (this.filebrowserSupportedImages.length > 0) {
+        const isNotificationVisible = localStorage.getItem('backendaiwebui.filebrowserNotification');
+        if ((isNotificationVisible == null || isNotificationVisible === 'true') && !this.isWritable) {
+          this.shadowRoot.querySelector('#filebrowser-notificatin-dialog').show();
```
Comment (adrysn) [inline src/components/backend-ai-storage-list.ts]: same here.

**Review by adrysn — APPROVED** [review]
LGTM except minor typo in the element id. After fixing them, I think this PR can be merged.

File: src/components/backend-ai-storage-list.ts (hunk 1 above)
Comment (lizable) [reply src/components/backend-ai-storage-list.ts]: Oops, I'll fix it right away.

**Comment by achimnol** [comment]
Resolved lablup/backend.ai#414.

**Pull request merged**